CONTRIBUTING.md

```diff
@@ -1,17 +1,39 @@
-# Contributing to the community repo
+# Contributing to the Community Repo
 
-Make a [pull request](https://help.github.com/articles/using-pull-requests) (PR).
+Welcome to the Kubernetes Community contributing guide. We are excited about the prospect of you joining our [community](https://github.com/kubernetes/community]!
 
-Upon successful review, someone will give the PR
-a __LGTM__ (_looks good to me_) in the review thread.
+## Finding Something to Work On
+
+Before you begin, you can look at some places where you can help out:
+
+- [Existing open issues](https://github.com/kubernetes/community/issues)
+- [Open Pull Requests](https://github.com/kubernetes/community/pulls) - Most of the pull requests in this repository are not code, but documentation and process. This is a great place to get started. Even if you do not have the rights to merge you can still help by doing a review.
 
-A [SIG lead](sig-list.md) (or someone with approval powers
+## Steps
+
+1. Make a [pull request](https://help.github.com/articles/using-pull-requests) (PR).
```

Comment:
s/pull request/Pull Request ?
Since we list its abbrev here, we can use `PR` instead of `Pull Request` in the below text, such as L24, L26